src/hooks/useWebR.tsx

```diff
@@ -231,7 +231,11 @@ webr::shim_install()
     filename <- file.path(appDir, name)
     path <- dirname(filename)
     dir.create(path, recursive = TRUE, showWarnings = FALSE)
-    writeLines(files[[name]], filename)
+    if (is.character(files[[name]])) {
+      writeLines(files[[name]], filename)
```

Comment:
I think the native encoding under Emscripten is UTF-8, so it might not matter here, but it makes sense to take the possibility of re-encoding out of the equation entirely with `useBytes = TRUE`, so I'll add that.